File: CozyHavenStayServer/CozyHavenStayServer/Models/DTO/ResetPasswordDTO.cs
namespace CozyHavenStayServer.Models.DTO
{
    public class ResetPasswordDTO
    {
        public string Email { get; set; }
        public string Password { get; set; }
        public string ConfirmPassword { get; set; }
        public string Token { get; set; }
    }
}

File: CozyHeavenStay/src/app/models/DTO/ResetPasswordDTO.ts
export class ResetPasswordDTO {
    email: string;
    password: string;
    confirmPassword: string;
    token: string;
  
    constructor(email: string, password: string, confirmPassword: string, token: string) {
      this.email = email;
      this.password = password;
      this.confirmPassword = confirmPassword;
      this.token = token;
    }
  }